Portfolio.html

```diff
@@ -0,0 +1,59 @@
+<!DOCTYPE html>
+
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="css/myStyle.css"/>
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css">
+    <title>Portfolio</title>
+  </head>
+
+
+  <body>
+    <main class="content">
+      <header class="header-container">
+          <div class="Logo">
+            <img src="Logo.png" alt="log"/>
+          </div>
+
+          <div class="hamburger-btn">
+            <i class="fa fa-bars"></i>
+          </div>
+      </header>
+    
+      <div class="headline">
+        <div class="intro-message">
+          <h1 class="intro_1">
+            I’m Bonke
+          </h1>
+          <h1 class="intro_2">
+            Glad To see you!
+          </h1>
+        </div>
+        
+        <div class="headline-text">
+            <p>
+                I’m a software developer! I can help you build a product , 
+                feature or website Look through some of my work and experience! 
+                If you like what you see and have a project you need coded, 
+                don’t hestiate to contact me.
+            </p>
+        </div>
+        
+        
+        <div class="connect_link">
+          <a href="#">LET'S CONNECT</a>
+        </div> 
+        
+        <div class="social_media_icons">
+            <div class="twitter"><a href="#"><img src="twitter.png" alt="twitter_bird"/></a></div>
+            <div class="M"><a href="#"><img src="M_icon.jpg" alt="M_picture"/></a></div>
+            <div class="linkedIn"><a href="#"><img src="Linkedin.jpg" alt="linkedIn_icon"/></a></div>
+            <div class="git"><a href="#"><img src="github.jpg" alt="github_icon"/></a></div>
+            <div class="hand"><a href="#"><img src="hand_icon.jpg" alt="hand"/></a></div>    
+        </div>
```

Comment:
- [ ] Please ensure to wrap the icons inside the `<ul>` tag and each icon inside the `<li>` tag. Also, ensure that each and all icons are links. This can be achieved by wrapping them inside the `<a>` tag.